tests/tests_bridge.yml

```diff
@@ -27,24 +40,40 @@
     - linux-system-roles.network
 
 - import_playbook: run-tasks.yml
+  tags:
+    - 'tests::tier1'
   vars:
     task: tasks/assert-device_present.yml
 
 - import_playbook: run-tasks.yml
+  tags:
+    - 'tests::tier1'
   vars:
     profile: "{{ interface }}"
     task: tasks/assert-profile_present.yml
 
 - import_playbook: down-profile.yml
+  tags:
+    - 'tests::tier1'
+    - 'tests::cleanup'
+    - 'tests::tier1::cleanup'
+    - 'tests::net::bridge::cleanup'
   vars:
     profile: "{{ interface }}"
 # FIXME: assert profile/device down
 
 - import_playbook: remove-profile.yml
+  tags:
+    - 'tests::tier1'
+    - 'tests::cleanup'
+    - 'tests::tier1::cleanup'
+    - 'tests::net::bridge::cleanup'
```

Comment:
Those two plays above should not be tagged as cleanups. They are integral part of the test, given that we assert the profile absence below. If we skip them, the test will fail.